test/support/child_crash_test/pipeline.ex

```diff
@@ -92,4 +108,9 @@ defmodule Membrane.Support.ChildCrashTest.Pipeline do
 
     send(pid, {:create_path, spec})
   end
+
+  @spec inform_about_details_in_case_of_crash(pid(), pid()) :: any()
+  def inform_about_details_in_case_of_crash(pid, send_to) do
+    send(pid, {:inform_about_crash, send_to})
```

Comment:
```suggestion
    send(pipeline, {:inform_about_crash, self()})
```

The only value, that is passed as `send_to`, is `self()`